apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -1223,10 +1225,34 @@ export class PostgresClient extends BasicDatabaseClient<QueryResult> {
   }
 
   protected async configDatabase(server: IDbConnectionServer, database: { database: string}) {
+
+    let resolvedPw = null;
+    const redshiftOptions = server.config.redshiftOptions;
+    console.log("redshiftOptions", redshiftOptions);
+    if (
```

Comment:
Seems like this change only adds support for credentials stored in a credentials file, is that correct?

Reply:
That is correct, I need to investigate the CLI auth, that WILL be part of this PR unless you want it in a seperate PR, but if it's anything like Azure CLI, I can't see it being a huge change.

Reply:
Let's do a separate PR